### PR TITLE
Added md5sum plugin

### PR DESCRIPTION
**Pull request by ath3**
Plugin to create and verify md5 checksums

### CODE REVIEW

File: plugins/md5sum [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+resp=f
+
+if [ -s "$selection" ]; then
+    echo -n "work with selection (s) or current file (f) [default=f]: "
+    read resp
+fi
+
+if [ "$resp" = "s" ]; then
+    arr=$(cat $selection | tr '\0' '\n')
+    { read -r file; } <<< "$arr"
```
Comment (jarun) [inline plugins/md5sum]: This is Bash, not sh.

File: plugins/md5sum [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+resp=f
+
+if [ -s "$selection" ]; then
+    echo -n "work with selection (s) or current file (f) [default=f]: "
+    read resp
+fi
+
+if [ "$resp" = "s" ]; then
+    file=$(basename "$(cat $selection | tr '\0' '\n' | head -n 1)").md5
+    cat "$selection" | sed 's|'"$PWD/"'||g' | xargs -0 -i md5sum {} > "$file"
```
Comment (jarun) [inline plugins/md5sum]: You know shell scripts much better than me! :+1: 

Care to modify ndiff and splitjoin plugins to be POSIX-compliant? I would really appreciate that.

**Comment by jarun** [comment]
I would also suggest to rename the plugin to `checksum` and ask the user whether to calculate md5 or sha256 checksum. sha256 is quite popular and would help a lot of users (including me ;)).

**Comment by ath3** [comment]
I removed the "-o" parameter from xargs, because with that parameter this plugin was not working for me.
This means that we will get the warning message "Input is not from a terminal", but it works.

Since POSIX doesnt have process substitution, the directory comparison now lists the directory contents to two temporary files, and those files will be compared.

**Comment by jarun** [comment]
> ndiff

No problem! Thank you!

File: plugins/checksum [hunk 3]
```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env sh
+
+# Description: Create and verify checksums
+#
+# If selection is used: it will generate one file containing the checksums with file names
+#                       [and with paths if they are in another directory]
+#                       The filename will be checksum_timestamp.checksum_type
+# If file is used: if the file is a checksum, it does the verification
+#                  if the file is not a checksum, it will be created from it
```
Comment (jarun) [inline plugins/checksum]: "it will be created from it" - @ath3 what does this mean?

Comment (jarun) [reply plugins/checksum]: OK. Got it!

**Pull request merged**